### PR TITLE
UI: Mobile Views

### PR DESCRIPTION
**Pull request by DingoEatingFuzz**
**Builds on #4533, rebase after that merges**

I would call these "mobile acceptable", not "mobile friendly". 

We still don't prioritize using the UI on a phone, but hey, sometimes you gotta do what you gotta do.

image

image

image

image

The breakpoint for the flyout menu is at 960px, so it will also help people who don't use their whole screen for their browser.

image

### CODE REVIEW

File: ui/app/styles/components/gutter-toggle.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.gutter-toggle {
+  display: none;
+
+  position: absolute;
+  left: 0;
+  padding: 1rem;
+  transform: translate(-35%, 0%);
+  height: 100%;
+  fill: desaturate(lighten($nomad-green, 20%), 30%);
+  cursor: pointer;
```
Comment (joshuaogle) [inline ui/app/styles/components/gutter-toggle.scss]: Any thoughts on using a style more similar to Terraform's icon they use here? The usage is different, but I like that the icon is pretty minimal

image

File: ui/app/styles/components/gutter-toggle.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.gutter-toggle {
+  display: none;
+
+  position: absolute;
+  left: 0;
+  padding: 1rem;
+  transform: translate(-35%, 0%);
+  height: 100%;
+  fill: desaturate(lighten($nomad-green, 20%), 30%);
```
Comment (joshuaogle) [inline ui/app/styles/components/gutter-toggle.scss]: It'll be really nice to have the full palette soon :\

File: ui/app/styles/components/gutter.scss [hunk 3]
```diff
@@ -1,4 +1,76 @@
...
+
+  .collapsed-menu {
+    display: none;
+    position: relative;
+    height: 3.5rem;
+
+    width: $gutter-width;
+    transform: translateX($gutter-width);
+    z-index: $z-gutter;
+
```
Comment (joshuaogle) [inline ui/app/styles/components/gutter.scss]: Are these spaces intentional?

Comment (DingoEatingFuzz) [reply ui/app/styles/components/gutter.scss]: Yes and no? I remember thinking it made the code read better, particularly the part where the display is `none` despite all those other styles being there. 

Looking at it again now, I don't find it helpful. Will remove.

**Comment by shantanugadgil** [comment]
_In my opinion, anything "acceptable" 🙌  is better that "totally does not work"; for that late night call when you have to debug something serious! 😁_

**Pull request merged**